### PR TITLE
[ADD] button to the form to go full width + styling

### PR DESCRIPTION
**Pull request by Simon-Busch**
- [x] Update form component to handle expand state
- [x] Update styling
- [x] Change text to icons

### CODE REVIEW

File: src/components/reporter/Form.module.scss [hunk 1]
```diff
@@ -46,3 +58,4 @@
     }
   }
 }
+
```
Comment (joshuashort) [inline src/components/reporter/Form.module.scss:61]: Not an extra newline, just ending with one

```suggestion
```

File: src/components/reporter/Form.js [hunk 2]
```diff
@@ -482,3 +492,4 @@ const Form = ({ contest, sponsor, repoUrl }) => {
 };
 
 export default Form;
+
```
Comment (joshuashort) [inline src/components/reporter/Form.js:495]: Same as below

```suggestion
```

File: src/components/reporter/Form.js [hunk 3]
```diff
@@ -318,8 +319,17 @@ const Form = ({ contest, sponsor, repoUrl }) => {
...
-            <h1>{sponsor} contest finding</h1>
+          <div className={!isExpended ? clsx(styles.Form) : clsx(styles.Form, styles.FormMax)}>
+            <div className={clsx(styles.FormHeader)}>
+              <h1>{sponsor} contest finding</h1>
+              <button
+                className="button cta-button"
+                type="button"
+                onClick={() => setIsExpended(!isExpended)}
+              >
+                {isExpended ? 'Reduce' : 'Expend'}
```
Comment (joshuashort) [inline src/components/reporter/Form.js:330]: These are not the right words..

I'm struggling to come up with a decent pair, but 'Expend' should definitely be more like 'Expand' or 'Extend'

Comment (Simon-Busch) [reply src/components/reporter/Form.js:330]: As discussed, will go with icons instead

Comment (joshuashort) [reply src/components/reporter/Form.js:330]: For the code, go with 'isExpanded'; icons won't save us there.

Comment (JustDravee) [reply src/components/reporter/Form.js]: Wow that was fast. Thanks team!

**Comment by Simon-Busch** [comment]
@tealtigrrr  can you check if the icon choice is OK for you ?

File: src/components/reporter/Form.js [hunk 4]
```diff
@@ -318,8 +319,31 @@ const Form = ({ contest, sponsor, repoUrl }) => {
...
+                    onClick={() => setIsExpanded(false)}
+                  />
+                ) : (
+                  <img
+                    src="/images/expand.svg"
+                    alt="expand form"
+                    className={clsx(styles.FormIcons)}
+                    onClick={() => setIsExpanded(true)}
+                  />
+                )}
```
Comment (joshuashort) [inline src/components/reporter/Form.js:345]: While we're waiting on icon approval.. what do you think of this form?

```suggestion
                <img
                  src={isExpanded ? "/images/compress.svg" : "/images/expand.svg"}
                  alt={isExpanded ? "compress form" : "expand form"}
                  className={clsx(styles.FormIcons)}
                  onClick={() => setIsExpanded(!isExpanded)}
                />
```

Comment (Simon-Busch) [reply src/components/reporter/Form.js]: Much better indeed :)

**Review by joshuashort — APPROVED** [review]

**Pull request merged**